pkg/sdk/common_types.go

```diff
@@ -57,6 +57,12 @@ type In struct {
 	Schema   DatabaseObjectIdentifier `ddl:"identifier" sql:"SCHEMA"`
 }
 
+type ExtendedIn struct {
```

Comment:
nit: This name is too generic but TBH I don't have a better name for it now, we can change it later if needed.

Reply:
Maybe it is, but I wanted to avoid a name like `InWithApplicationAndApplicationPackage` for now.